sub-packages/bionemo-geneformer/tests/bionemo/geneformer/scripts/test_train_geneformer.py

```diff
@@ -56,49 +70,90 @@ def test_main_runs(tmpdir, create_checkpoint_callback: bool, data_path: Path):
             result_dir=result_dir,
             wandb_project=None,
             wandb_offline=True,
-            num_steps=5,
-            limit_val_batches=1,
-            val_check_interval=2,
+            num_steps=20,  # Run for more steps to ensure metrics are logged
```

Comment:
can we change it back to smaller number of step? why this change? and lets limit val batches to 1